server/webapp/WEB-INF/rails.new/spec/new_javascripts/views/pipeline_configs/pipeline_stage_field_widget_spec.js

```diff
@@ -48,7 +53,7 @@ define([
 
         mount(material, Pipelines);
 
-        expect(_.isEmpty($root.find("input[name='pipeline-stage']").val())).toBe(true);
+        expect($root.find("input[name='pipeline-stage']")).toBeEmpty();
```

Comment:
Please understand what `toBeEmpty` does https://api.jquery.com/empty-selector/

What you probably intend to do is `.toHaveValue('')`